packages/ts/generator-typescript-plugin-backbone/src/EndpointMethodRequestBodyProcessor.ts

```diff
@@ -18,54 +18,88 @@ export type EndpointMethodRequestBody = ReadonlyDeep<OpenAPIV3.RequestBodyObject
 export type EndpointMethodRequestBodyProcessingResult = Readonly<{
   parameters: readonly ParameterDeclaration[];
   packedParameters?: ObjectLiteralExpression;
+  initParam: ts.Identifier;
 }>;
 
 export default class EndpointMethodRequestBodyProcessor {
+  static readonly #defaultInitParamName = 'init';
```

Comment:
```suggestion
  static readonly #defaultInitParamName = '__init';
```

Reply:
That works differently in the multi-module generator: the default name is `init` and is modified if there's another parameter with the same name, so there's no need to add the underscores by default.